client/components/DisplayTest/index.jsx

```diff
@@ -42,27 +46,10 @@ class DisplayTest extends Component {
         this.handleModalCloseClick = this.handleModalCloseClick.bind(this);
         this.handleModalConfirmClick = this.handleModalConfirmClick.bind(this);
 
-        this.testIframe = React.createRef();
-    }
+        // iframe action
+        this.handleSubmit = this.handleSubmit.bind(this);
 
-    handleRaiseIssueClick() {
```

Comment:
@spectranaut I reordered a few things while rebasing in order to keep myself sane. this looks like a worse refactor than it is, but these are still here just under the other click handlers.